### PR TITLE
Unit test statistical

### PR DESCRIPTION
**Pull request by mvn-haphan-dn**
## Description
A few sentences describing the overall goals of the pull request's commits.
## Reference Requirements
- Task ID: #74 

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: tests/Browser/Pages/Backend/HomeTest.php [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    {
+        $languages = ['English', 'VietNamese'];
+        parent::setUp();
+        
+        factory(Category::class, self::NUMBER_RECORD_CREATE)->create();
+        factory(Book::class, self::NUMBER_RECORD_CREATE)->create();
+        factory(User::class, self::NUMBER_RECORD_CREATE)->create();
+        factory(Post::class, self::NUMBER_RECORD_CREATE)->create();
+        factory(Borrow::class, self::NUMBER_RECORD_CREATE)->create();
+        $borrowId = DB::table('borrowes')->pluck('id')->all();
```
Comment (at-haytran) [inline tests/Browser/Pages/Backend/HomeTest.php]: sửa thành số nhiều borrowIds

File: tests/Browser/Pages/Backend/HomeTest.php [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+     * Test URL Home Page.
+     *
+     * @return void
+     */
+    public function testURLHomePage()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->loginAs($this->user)
+                ->visit('/admin')
+                ->clicklink('AdminLTE')
```
Comment (at-haytran) [inline tests/Browser/Pages/Backend/HomeTest.php]: clickLink

**Comment by at-haytran** [comment]
LGTM

**Comment by at-huydq** [comment]
LGTM

**Pull request merged**